submissions/SkirnevskyiAleksandr/HTML-CSS-Popup/style.css

```diff
@@ -0,0 +1,158 @@
+* {
+  font-size: 14px;
+}
+img {
```

Comment:
could you please add the class for image and add styles from a tag img to this class? (it is a good practice only)